spec/views/content_items/show.html.erb_spec.rb

```diff
@@ -3,11 +3,13 @@
 RSpec.describe 'content_items/show.html.erb', type: :view do
   let(:content_item) { build(:content_item).decorate }
   let(:organisation) { build(:organisation) }
+  let(:taxonomy) { build(:taxonomy, title: "taxon title") }
 
   before do
     assign(:content_item, content_item)
     assign(:organisation, organisation)
     content_item.organisations << organisation
+    content_item.taxonomies << taxonomy
```

Comment:
Do we need to add the taxonomies in this block? 
The `<<` is persisting the taxonomies, BTW